app/css/components/StockTickerItem.css

```diff
@@ -0,0 +1,17 @@
+.stock-ticker-item {
+  display: inline;
+  margin-right: 15px;
+  font-size: 1.3em;
+}
+
+.stock-ticker-item > span {
+  margin-right: 5px;
+}
+
+.stock-ticker-item > .sti-change-positive {
+  color: #00FF00;
+}
+
+.stock-ticker-item > .sti-change-negative {
+  color: #FF0000;
+}
```

Comment:
Yeah, `gulp` is configured to recursively find and concatenate css files:
https://github.com/TeamMartinez/YOLO-Client/blob/stock-ticker/gulp/config.js#L13